docs/quickstart/views/standalone-assessment.ejs

```diff
@@ -3,7 +3,7 @@
 <head lang="en">
     <meta charset="UTF-8">
     <title>Learnosity SDK - NodeJS</title>
-    <script src="//items.learnosity.com/?v2022.2.LTS"></script>
+    <script src="//items.learnosity.com/?latest"></script>
```

Comment:
@bhavya-shukla-lrn I think we should use "latest-lts" here, because I really wanted to emphasise the need to use an LTS version in production. Agree that the static version needed too much updating for maintenance.